packages/ag-charts-enterprise/src/axes/radius-category/radiusCrossLine.ts

```diff
@@ -0,0 +1,157 @@
+import { _ModuleSupport, _Scale, _Scene, _Util } from 'ag-charts-community';
+import { PolarCrossLine, PolarCrossLineLabel } from '../polarCrossLine';
+
+const { ChartAxisDirection, Validate, OPT_NUMBER } = _ModuleSupport;
+const { Path, Sector, Text } = _Scene;
+const { normalizeAngle360, toRadians, isNumberEqual } = _Util;
+
+class RadiusCrossLineLabel extends PolarCrossLineLabel {
+    @Validate(OPT_NUMBER(-360, 360))
+    positionAngle?: number = undefined;
+}
+
+export class RadiusCrossLine extends PolarCrossLine {
```

Comment:
This seems to be an identical implementation to the `radius-number` axis implementation? Can we factor out the common code please?